lib/travis/build/data.rb

```diff
@@ -105,6 +112,15 @@ def disable_sudo?
         !!data[:paranoid]
       end
 
+      def default_ruby
+        [config[:os], config[:dist]].each do |ruby_key|
```

Comment:
agreed.  an iteration of this had 4 keys with some string manipulation, but with only two + default I think `||` is fine